python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/aaa-accounting.j2

```diff
@@ -6,48 +6,50 @@
 {# eos - aaa accounting #}
 {% if aaa_accounting is arista.avd.defined %}
 {%     if aaa_accounting.exec.console.type is arista.avd.defined %}
-{%         set aaa_accounting_exec_console_cli = "aaa accounting exec console" %}
 {%         if aaa_accounting.exec.console.type is arista.avd.defined('none') %}
-{%             set aaa_accounting_exec_console_cli = aaa_accounting_exec_console_cli ~ " none" %}
-{%         else %}
-{%             set aaa_accounting_exec_console_cli = aaa_accounting_exec_console_cli ~ " " ~ aaa_accounting.exec.console.type %}
+aaa accounting exec console none
+{%         elif aaa_accounting.exec.console.group is arista.avd.defined or aaa_accounting.exec.console.logging is arista.avd.defined(true) %}
+{%             set exec_console_cli = "aaa accounting exec console " ~ aaa_accounting.exec.console.type %}
```

Comment:
I think we should have a check for `aaa_accounting.exec.console.type` defined before this statement, what if `type` is not defined but `group` and `logging` are defined.